pkg/webhook/mutatingwebhook.go

```diff
@@ -174,3 +191,53 @@ func parseSidecarContainerImage(pod *corev1.Pod) (string, error) {
 
 	return image, nil
 }
+
+func (si *SidecarInjector) supportsNativeSidecar() (bool, error) {
+	list, err := si.NodeLister.List(labels.Everything())
+	if err != nil {
+		return false, fmt.Errorf("failed to get cluster nodes: %w", err)
+	}
+
+	supportsNativeSidecar := true
+	for _, node := range list {
+		version, err := getRelease(node.Status.NodeInfo.KubeletVersion)
+		if version < "1.29" || err != nil {
```

Comment:
Using string comparison to compare major versions. Unit tests work fine, but I'm wondering if there's a better way...